aws-v1/client/client.go

```diff
@@ -12,11 +12,11 @@ import (
 	"github.com/aws/aws-sdk-go/aws/request"
 	"github.com/aws/aws-sdk-go/service/dynamodb"
 	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbiface"
-	"github.com/truora/minidyn/interpreter"
+	"github.com/ldelacruztruora/minidyn/core"
```

Comment:
are this paths correct?

Reply:
no it was not. I updated the go.mod with the correct module name